### DISCUSSION TITLE
请问metacubexd面板怎么让他显示emoji呢

### DISCUSSION THREAD

**Discussion by refined-fish** [nikkinikki-org/OpenWrt-nikki#326] [Q&A] [answered]
之前在用OpenClash，metacubexd面板中能显示节点前的小国旗emoji，但是现在就不显示了，只显示字母HK之类的

**Comment by morytyann** [comment, marked as answer]
![image](https://github.com/user-attachments/assets/659afa1d-b14c-41e1-a795-d18d6e93dc72)

**Reply by refined-fish** [reply]
爱你❤️
